nilearn/plotting/surface/tests/test_matplotlib_backend.py

```diff
@@ -492,20 +545,24 @@ def test_plot_surf_roi_matplotlib_specific_plot_to_axes(
 def test_plot_surf_contours_fig_axes(
     matplotlib_pyplot, in_memory_mesh, parcellation
 ):
-    fig, axes = plt.subplots(1, 1, subplot_kw={"projection": "3d"})
+    fig, axes = matplotlib_pyplot.subplots(
+        1, 1, subplot_kw={"projection": "3d"}
+    )
     plot_surf_contours(in_memory_mesh, parcellation, axes=axes)
     plot_surf_contours(in_memory_mesh, parcellation, figure=fig)
 
 
-def test_plot_surf_contours_error(rng, in_memory_mesh, parcellation):
+def test_plot_surf_contours_error(
+    matplotlib_pyplot, rng, in_memory_mesh, parcellation
+):
     # we need an invalid parcellation for testing
```

Comment:
Small docstring here ?

Reply:
I haven't put docstrings for functions other than `plot_surf`. I'll write them down when I refactor for those functions. Actually I had put a note for that https://github.com/nilearn/nilearn/pull/5322#issuecomment-2794321270.